src/js/models/Course.js

```diff
@@ -50,6 +50,10 @@ export default class Course {
         this.#creditHour = calculate_credit_hours(this.#name);
     }
 
+    toJson () {
```

Comment:
```suggestion
    toObject () {
```
This might be better named `toObject` since it's not actually returning json.